### ISSUE TITLE
[A11y] Dashboard - Reflow and Mobile - Graphs

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1857]
### Operating System

Windows

### Browser

Edge

### Affected page

Backend

### Admin user?

Yes

### Issue Description

At 1280x1024 and 400% zoom, you have to scroll horizontally to get to some of the content.  The same is true on a fairly standard view on my phone.  The text and tables also get really difficult / impossible to view.

Here are two screenshots from a resized desktop window which I managed to make look a lot like my phone's view:

First, you can see the navigation menu scrolled slightly out of view, and the text still being cut off horizontally:

![image](https://user-images.githubusercontent.com/2157974/128935042-bfa4b84c-9168-4cff-9a68-95ecd1a75ae1.png)

Then the extremely broken graphs:

![image](https://user-images.githubusercontent.com/2157974/128935099-e953a523-297e-47ec-9e0e-f2a08a0d0317.png)

### Suggested Fix

A huge part of the problem is the admin menu.  When it's expanded, it pushes everything to the right, even things that aren't displayed until well after its box ended.  If that menu were to stack vertically, most of these problems likely go away.  At least in my testing, when the menu is put into the smallest state, I can use my phone a lot better.  If it weren't taking up *any* horizontal real estate, I think everything would work well for both reflow and mobile use.

**Comment by jechols** [comment]
Because this issue happens on the ultra-zoomed-in view pretty much the same as on a phone, it should be something anybody can QA just by looking at the page on a phone.

**Comment by azle** [comment]
Updated 1/6/2024

Still a current issue.
